isso/js/app/i18n.js

```diff
@@ -2,8 +2,8 @@ define(["app/config", "app/i18n/bg", "app/i18n/cs", "app/i18n/de",
         "app/i18n/en", "app/i18n/fa", "app/i18n/fi", "app/i18n/fr",
         "app/i18n/hr", "app/i18n/ru", "app/i18n/it", "app/i18n/eo",
         "app/i18n/sv", "app/i18n/nl", "app/i18n/el_GR", "app/i18n/es",
-        "app/i18n/vi", "app/i18n/zh_CN"],
-        function(config, bg, cs, de, en, fa, fi, fr, hr, ru, it, eo, sv, nl, el, es, vi, zh) {
+        "app/i18n/vi", "app/i18n/zh_CN", "app/i18n/zh_CN", "app/i18n/zh_TW"],
```

Comment:
Actually I'd like to make configuration items "zh"," zh_CN", "zh_TW" correspond to the files "app/i18n/zh_CN", "app/i18n/zh_CN", "app/i18n/zh_TW", respectively (if I understood it right). In other words, treat the "zh" as an alias of "zh_CN".